hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -108,10 +108,12 @@ class HassioAddonConfig extends LitElement {
 
   protected firstUpdated(changedProps) {
     super.firstUpdated(changedProps);
-    this._canShowSchema = !this.addon.schema.find(
-      // @ts-ignore
-      (entry) => !SUPPORTED_UI_TYPES.includes(entry.type) || entry.multiple
-    );
+    this._canShowSchema =
+      this.addon.schema != null &&
```

Comment:
```suggestion
      this.addon.schema !== null &&
```

Reply:
That's what I get when jumping between python and TS :D